src/acceptance-tests-brain/test-scripts/013_credhub_test.rb

```diff
@@ -8,8 +8,18 @@
 CH_CLI = 'credhub'
 CH_SERVICE = "https://credhub.#{ENV['CF_DOMAIN']}"
 
-# Check if credhub is running, otherwise skip the test.
-status = run_with_status('curl', '--silent', '--fail', '--insecure', "#{CH_SERVICE}/info")
+# Check if credhub is running, otherwise skip the test.  We do not
+# break on the first failure, but check a bit more, in case it was a
+# transient issue with the endpoint. We do accept sucess immediately,
+# tough.
```

Comment:
```suggestion
# though.
```
… though it _is_ to amusing to think that you mean "we accept it, whether you like it or not" ;)

Reply:
Heh. Will fix. The `sucess` above as well, should `success`.

Reply:
Addressed by commit f2be8c95181e3a6e5e099501d59df6ae6a4d7c48